src/ontology/mondo-ingest.Makefile

```diff
@@ -664,14 +664,19 @@ $(EXTERNAL_CONTENT_DIR)/nando-mappings.robot.tsv: $(MAPPINGSDIR)/mondo-nando.sss
 
 # OTAR / EFO
 .PHONY: external-content-efo
-external-content-efo: $(EXTERNAL_CONTENT_DIR)/mondo-efo.robot.owl $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.owl
+external-content-efo: $(EXTERNAL_CONTENT_DIR)/mondo-efo.robot.owl $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.owl $(EXTERNAL_CONTENT_DIR)/efo-proxy-merges.robot.owl
 
 $(EXTERNAL_CONTENT_DIR)/mondo-efo.robot.tsv:
 	wget "https://raw.githubusercontent.com/EBISPOT/efo/master/src/ontology/reports/mondo-efo.robot.tsv" -O $@
 
 $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.tsv:
 	wget "https://raw.githubusercontent.com/EBISPOT/efo/master/src/ontology/reports/mondo-otar-subset.robot.tsv" -O $@
 
+# Unfortunately, @matentzn cant find the orginal TSV file, but it is highly unlikely this file will ever change.
```

Comment:
This comment is above `efo-proxy-merges.robot.owl`, but it says TSV; Should it be on the line above `mondo-efo.robot.tsv` or `mondo-otar-subset.robot.tsv`?

Reply:
No I meant: the TSV file _corresponding_ to the `efo-proxy-merges.robot.owl` file. There must have been one in distant past (in externally managed content, all primary data is in TSV format, and OWL is derived automatically with robot. That file is checked in in OWL).